src/components/input/keyboardnavigation.js

```diff
@@ -145,6 +145,32 @@ define(["inputManager", "layoutManager"], function (inputManager, layoutManager)
             }
         });
     }
+    // For hiding the mouse while playbackç
+    require(["components/input/mouseManager"]);
```

Comment:
why is this component not present in `site.js`?

Reply:
My PR was published before many modules were moved to ``site.js`` I think. Moving this now.